hack/build/ci/create-manifest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -z "$2" ]
+then
+  echo "Usage: $0 <image_name> <image_tag> <enable_multiplatform>"
+  exit 1
+fi
+
+image_name=$1
+image_tag=$2
+multiplatform=$3
+image="${image_name}:${image_tag}"
+
+if [ "$multiplatform" == "true" ]
+then
+  supported_architectures=("amd64" "arm64" "ppc64le" "s390x")
+  images=()
+  echo "Creating manifest for ${supported_architectures[*]}"
+
+  for architecture in "${supported_architectures[@]}"
+  do
+    docker pull "${image}-${architecture}"
```

Comment:
can we switch to `podman` here? easiest way to get the manifest in oci format instead of docker

Reply:
seems `podman-login` action exists: https://github.com/marketplace/actions/podman-login